airflow/README.md

```diff
@@ -38,7 +40,7 @@ Edit the `airflow.d/conf.yaml` file, in the `conf.d/` folder at the root of your
    ```conf
    [scheduler]
    statsd_on = True
-   statsd_host = localhost
+   statsd_host = localhost # or <VM_IP_address>
    statsd_port = 8125
```

Comment:
```suggestion
   statsd_host = localhost  # Hostname or IP of server running the Datadog Agent
   statsd_port = 8125       # DogStatsD port configured in the Datadog Agent
```
This might be more precise ?